satellite/metainfo/metainfo.go

```diff
@@ -702,19 +707,55 @@ func (endpoint *Endpoint) CreateBucket(ctx context.Context, req *pb.BucketCreate
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
-	bucket, err := convertProtoToBucket(req, keyInfo.ProjectID)
-	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, err.Error())
-	}
+	bucket, err := endpoint.metainfo.GetBucket(ctx, req.GetName(), keyInfo.ProjectID)
+	if err == nil {
```

Comment:
if you are going to keep this update functionality in the create method, can you at least add a comment explaining what this block is about

Reply:
added the comment briefing the logic